programs/us/us.cpp

```diff
@@ -644,82 +723,223 @@ void US_Win::closeEvent( QCloseEvent* e )
+               Qt::AlignHCenter | Qt::AlignTop, builtLine);
+
+    // ---- Authors block ------------------------------------------------------
+    y = yVB + hVB + blockGap + authorsLift;
+
+    p.setPen(textHi);
+    p.setFont(authorsHdrFont);
+    p.drawText(QRect(margin, y, canvasW - 2*margin, hAuthorsHdr),
+               Qt::AlignHCenter | Qt::AlignTop, QObject::tr("Authors:"));
+    y += hAuthorsHdr + authorsGap;
+
+    p.setPen(textLo);
+    p.setFont(authorsFont);
+    for (int i = 0; i < authors.size(); ++i) {
+        p.drawText(QRect(margin, y, canvasW - 2*margin, fmA.height()),
+                   Qt::AlignHCenter | Qt::AlignTop, authors.at(i));
+        y += fmA.height();
+        if (i + 1 < authors.size()) y += namesGap;
+    }
+
+    p.end();
+    return canvas;
 }
 
 void US_Win::logo( int width )
 {
-  // Splash image
-  QPixmap rawpix = US_Images::getImage( US_Images::US3_SPLASH );
-
-  int ph = rawpix.height();
-  int pw = rawpix.width();
-
-  QPixmap  pixmap( pw, ph );
-  QPainter painter( &pixmap );
-
-  painter.drawPixmap( 0, 0, rawpix );
-  painter.setPen    ( QPen( Qt::white, 3 ) );
-
-  QString version = "Version " + US_Version + " ( " REVISION
-  " ) for " OS_TITLE;  // REVISION is #define "Revision: xxx"
-
-  QFont font( "Arial" );
-  font.setWeight( QFont::DemiBold );
-  font.setPixelSize( 16 );
-  painter.setFont( font );
-  QFontMetrics metrics( font );
-
-  int sWidth = metrics.boundingRect( version ).width();
-  int x      = ( pw - sWidth ) / 2;
-
-  painter.drawLine( 0, 106, pw, 106 );
-  painter.drawText( x, 132, version );
-  painter.drawLine( 0, 144, pw, 144 );
-
-  QString s = "Authors:";
-  sWidth    = metrics.boundingRect( s ).width();
-  painter.drawText( ( pw - sWidth ) / 2, 166, s );
-  s      = "Borries Demeler";
-  sWidth = metrics.boundingRect( s ).width();
-  painter.drawText( ( pw - sWidth ) / 2, 190, s );
-  s      = "Emre Brookes";
-  sWidth = metrics.boundingRect( s ).width();
-  painter.drawText( ( pw - sWidth ) / 2, 208, s );
-  s      = "Alexey Savelyev";
-  sWidth = metrics.boundingRect( s ).width();
-  painter.drawText( ( pw - sWidth ) / 2, 226, s );
-  s      = "Gary Gorbet";
-  sWidth = metrics.boundingRect( s ).width();
-  painter.drawText( ( pw - sWidth ) / 2, 244, s );
-  
-  smallframe = new QLabel(this);
-  smallframe->setPixmap(pixmap);
-  smallframe->setGeometry( (unsigned int)( (width / 2) - 230 ), 110, 460, 276);
+    // Splash (from resources via US_Images)
+    QPixmap raw = US_Images::getImage(US_Images::US3_SPLASH);
+
+    // Available area in central window (keeps your 710x532 layout)
+    const int yTop   = menuBar()->size().rheight();
+    const int availH = 532 - yTop - statusBar()->size().rheight();
+    const int availW = 710;
+    const int margin = 16;
+
+    // Strings (keep your existing content)
+    const QString version = QString("Version %1 (build %2)")
+            .arg(US_Version).arg(BUILD_NUMBER);
+    const QString builtOn = QString("Built on %1 at %2")
```

Comment:
The "built on" is for qmake accurate, but cmake would set the time to when the config `cmake ...` is run not when `cmake --build ...` is executed or? Would we care about the difference?